treetraversal.py

```
@@ -0,0 +1,129 @@
+    if isinstance(root, BaseComponent):
+        component = root
+        shadow_child = traverse_tree(component.render())
+        id = get_next_component_id()
+        shadow_node = ShadowNode(id, WidgetTypes.Component, component, subscribe_to_props_helper)
+        shadow_node.children = [shadow_child]
+        shadow_node.current_props = component.props
+        shadow_node.init()
+        return shadow_node
+    elif isinstance(root, WidgetNode):
+        id = get_next_widget_node_id()
+        raw_node = create_raw_childless_widget_node_with_id(id, root)
```

Ruff (0.8.2)

Local variable `raw_node` is assigned to but never used

Remove assignment to unused variable `raw_node`

(F841)